### PR TITLE
WIP: Update to Libhandy 1.0

### PR DESCRIPTION
**Pull request by tchx84**

### CODE REVIEW

**Comment by tchx84** [comment]
Current issue:

```
(com.github.tchx84.Flatseal:2): Gtk-CRITICAL **: 19:34:33.757: Error building template class 'FlatsealWindow' for an instance of type 'FlatsealWindow': .:185:1 Invalid object type 'HdyColumn'
```

Not clear indication, on the [migration guide](https://developer.puri.sm/projects/libhandy/unstable/hdy-migrating-0-0-to-1.html), how changes made to [HdyColumn](https://developer.puri.sm/projects/libhandy/unstable/HdyColumn.html) can cause this issue, given the [current way](https://github.com/tchx84/Flatseal/blob/master/src/widgets/window.ui#L187) I am using it (?).

**Comment by tchx84** [comment]
> Current issue:
> 
> ```
> (com.github.tchx84.Flatseal:2): Gtk-CRITICAL **: 19:34:33.757: Error building template class 'FlatsealWindow' for an instance of type 'FlatsealWindow': .:185:1 Invalid object type 'HdyColumn'
> ```
> 
> Not clear indication, on the [migration guide](https://developer.puri.sm/projects/libhandy/unstable/hdy-migrating-0-0-to-1.html), how changes made to [HdyColumn](https://developer.puri.sm/projects/libhandy/unstable/HdyColumn.html) can cause this issue, given the [current way](https://github.com/tchx84/Flatseal/blob/master/src/widgets/window.ui#L187) I am using it (?).

Hey @kekun any clue what could be the issue here? :)

NVM: just found out about https://gnome.pages.gitlab.gnome.org/libhandy/HdyClamp.html

**Comment by Kekun** [comment]
> NVM: just found out about https://gnome.pages.gitlab.gnome.org/libhandy/HdyClamp.html

See also https://gnome.pages.gitlab.gnome.org/libhandy/hdy-migrating-0-0-to-1.html :)

File: src/main.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ pkg.initFormat();
 pkg.require({
     Gio: '2.0',
     Gtk: '3.0',
-    Handy: '0.0',
+    Handy: '1',
```
Comment (AsciiWolf) [inline src/main.js:24]: _1_, not _1.0_?

Comment (Kekun) [reply src/main.js:24]: No because that's not the project version but the API version, 0.0 should have been 0 in the first place. :)

Comment (Kekun) [reply src/main.js:24]: I mean, it should have been 0 on libhandy's side when we started the project. regarding Flatseal, 0.0 was right and 1 is right. :)

**Pull request merged**